src/search/Search.js

```diff
@@ -58,6 +59,7 @@ class Search extends React.Component {
             <div className="Search">
                 <FavouriteAlertStripe />
                 <SavedSearchAlertStripe />
+                <UserAlertStripe />
```

Comment:
Hvis man er innlogget, men uten user, og følger direktelenke til /favoritter eller /lagrede-sok eller er inne på en annonse, så vil man der også få beskjed om å opprette bruker. Siden <UserAlertStripe />bare er lagt til på UserSettings.js og Search.js, så vil man ikke få alertstripen på andre sider enn disse to. Kan derfor heller legge <UserAlertStripe /> inn i Application.js og da vil den gjelde alle sider.

Reply:
👍 Fikser